README.md

```diff
@@ -692,7 +692,7 @@ ok, info, err := api.Auth.ValidateSession(nil, nil)
 assert.False(t, ok)
 assert.NotEmpty(t, info)
 assert.EqualValues(t, validateSessionResponse, info.JWT)
-assert.ErrorIs(t, err, errors.NoPublicKeyError)
```

Comment:
Why do we need to change this functions? wouldn't that still work?

Reply:
The builtin functions give better error messages

Reply:
You right, adjust all the errors back to use ErrorIs (Except specific tests for testing the "Is" function)